ngraph/core/src/pass/serialize.cpp

```diff
@@ -287,8 +287,6 @@ class XmlSerializer : public ngraph::AttributeVisitor {
         const std::vector<std::string>& result_mapping,
         pugi::xml_node& port_map,
         const std::string& portmap_name) {
-        NGRAPH_CHECK(!parameter_mapping.empty(), "No parameters found in body Function.");
```

Comment:
could you implement a new test for this case?

Reply:
yes, will be added